stage_tester/st.py

```diff
@@ -133,12 +169,23 @@ def main():
     if verbose:
         print("Auth settings:", auth)
 
+    if not (args.cluster_list or args.retrieve_results or args.compare_results):
+        print("No action requested, add -l, -r, or -c")
+        sys.exit(1)
+
     if args.cluster_list:
         retrieve_cluster_list(args.organization, args.address, proxies, auth, verbose)
 
     if args.retrieve_results:
         retrieve_results(args.address, proxies, auth, args.input, verbose)
 
+    if args.compare_results:
+        assert args.directory1 is not None, \
```

Comment:
I guess it could be better to assert directly `args.directory1`, because if the args.directory1 is defined, but empty, it shouldn't work